wandb/sdk/wandb_require.py

```diff
@@ -29,6 +30,12 @@ def __init__(self, features: Union[str, Sequence[str]]) -> None:
     def require_require(self) -> None:
         pass
 
+    def _require_concurrency(self) -> None:
+        os.environ["WANDB__CONCURRENCY"] = "True"
```

Comment:
Double underscore?  I wonder if "_WANDB_CONCURRENCY" would be better to indicate internal vars if that was the intent.

Reply:
yeah, it is ugly, just how the env parsing worked by default.
i changed it to WANDB_REQUIRE_CONCURRENCY